panther_lights/README.md

```diff
@@ -200,16 +206,18 @@ Arguments:
 
 Methods:
 
-- `__call__` - by default it is not implemented and must be overwritten. It returns an animation frame as a list of integers. It should also set the `self._finished` variable `true` when the last animation frame is returned.
-- `reset` - by default it is not implemented. It allows resetting animation.
+- `__call__` - It returns an animation frame as a list of integers, updates the `progress` variable, and sets the `finished` flag when the animation execution is finished. It uses the `_update_animation` method to get the current animation frame.
+- `reset` - It allows resetting animation to its initial state.
+- `_update_animation` - by default it is not implemented. This method is used to update the animation frame.
 
 Properties:
 
 - `brightness` [*int*]: returns animation brightness from `self._brightness`.
 - `num_led` [*int*]: returns number of LEDs in panel from `self._num_led`.
 - `finished` [*bool*]: returns if animation execution is finished from `self._finished`.
+- `progress` [*float*]: returns animation execution progress from `self._progress`.
```

Comment:
I think only `num_led` would be important for the end user. `brightness`, `finished` and `progress` are computed in parent class and are not left for the user to be part of the interface